styles.css

```diff
@@ -0,0 +1,3 @@
+@media screen {
+    
+}
```

Comment:
May you please remove the white space at the end so that your code can look nicer !!!